src/ssl.c

```diff
@@ -14501,7 +14501,9 @@ void SetupSession(WOLFSSL* ssl)
     }
 #endif
     session->timeout = ssl->timeout;
+#ifndef NO_ASN_TIME
```

Comment:
Did you check what bornOn is used for? Should it be set to 0 or something else??

Reply:
When `NO_ASN_TIME` was defined, it didn't look bornOn was used anywhere. References to bornOn are guarded by `NO_SESSION_CACHE`, which is defined when `NO_ASN_TIME` is defined. I should change the fix to be more explicit and use the appropriate macro. Will make the change shortly ....

Reply:
Done

Reply:
Don't you still need `NO_ASN_TIME` in the macro check also? `#if !defined(NO_SESSION_CACHE) && !defined(NO_ASN_TIME)`?

Reply:
Completely correct, I was being dumb. I reverted the last commit.` #if !defined(NO_ASN_TIME)` should be enough as `NO_ASN_TIME` defines `NO_SESSION_CACHE`